### PR TITLE
Best Practise: Fix TroveManagerTests with asset parameter

### PR DESCRIPTION
**Pull request by udosson**

### CODE REVIEW

**Comment by udosson** [comment]
All tests pass but `redeemCollateral(): reverts if fee eats up all returned collateral` that is listed in issue #188 

@wollooo - maybe you can help fixing the one `borrowerOperations`-related test

File: packages/contracts/contracts/ActivePool.sol [hunk 1]
```diff
@@ -23,36 +23,34 @@ import "./Dependencies/SafetyTransfer.sol";
...
-    address constant ETH_REF_ADDRESS = address(0);
+    string public constant NAME = "ActivePool";
 
     address public borrowerOperationsAddress;
     address public troveManagerAddress;
     address public stabilityPoolAddress;
     address public defaultPoolAddress;
-    uint256 internal ETH;  // deposited ether tracker
-    IDefaultPool public defaultPool;
+    uint256 internal ETH; // deposited ether tracker
```
Comment (wollooo) [inline packages/contracts/contracts/ActivePool.sol:37]: i think we can remove eth,  because we  dont  use it anymore

**Review by wollooo — COMMENTED** [review]
LGTM

File: packages/contracts/contracts/ActivePool.sol (hunk 1 above)
Comment (udosson) [reply packages/contracts/contracts/ActivePool.sol]: Agreed. Nice spot, @wollooo 
I removed ETH in an additional commit to this PR

**Pull request merged**